### DISCUSSION TITLE
Front end repo access

### DISCUSSION THREAD

**Discussion by colin-gannon** [f0cker/crackq#34] [General]
Hi,

Are there any plans to make the frontend for crackq available to extend/change? Tried reaching out to the maintainer (f0cker) but DMs are disabled everywhere!

Would love to make some minor changes and potentially help maintain the Vue.js part of the app.

Thanks!

**Comment by f0cker** [comment]
Hey, Yeah I can do that. I'll create the repo on Github on the weekend (it's a private repo in Gitlab atm), give me a nudge if you haven't heard anything by Sunday. I'm available on Discord if you need to reach me btw. 

Thanks

**Comment by colin-gannon** [comment]
Awesome! Thank you!

From: f0cker ***@***.***>
Date: Thursday, June 29, 2023 at 11:02 AM
To: f0cker/crackq ***@***.***>
Cc: Colin Gannon ***@***.***>, Author ***@***.***>
Subject: Re: [f0cker/crackq] Front end repo access (Discussion #34)
Hey, Yeah I can do that. I'll create the repo on Github on the weekend (it's a private repo in Gitlab atm), give me a nudge if you haven't heard anything by Sunday. I'm available on Discord if you need to reach me btw. Thanks —Reply to


Hey, Yeah I can do that. I'll create the repo on Github on the weekend (it's a private repo in Gitlab atm), give me a nudge if you haven't heard anything by Sunday. I'm available on Discord if you need to reach me btw.

Thanks

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/f0cker/crackq/discussions/34*discussioncomment-6316482__;Iw!!HWWtag!2n4B_YW7yQkzJ4FSeg_XQH1hjQWoLhemNhUcmuRV7rPB3r7XYbcROz3o_yGh7Q6txWlGZLWZHzhMWoH2FplixmI3$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/A3GSKXI3SS56YBWKPTVFA5TXNWKHVANCNFSM6AAAAAAZYRKORQ__;!!HWWtag!2n4B_YW7yQkzJ4FSeg_XQH1hjQWoLhemNhUcmuRV7rPB3r7XYbcROz3o_yGh7Q6txWlGZLWZHzhMWoH2FnzNjPB8$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

===============================================================================
Please refer to http://www.kkr.com/email-disclaimer for important disclosures regarding this electronic communication.
KKR uses your personal information in accordance with its privacy notices, which can be found on http://www.kkr.com.
===============================================================================

**Comment by f0cker** [comment]
This is done now, see here: https://github.com/f0cker/crackvue

I should have some time to start pushing some updates soon. Any problems give me a shout on the Discord channel.

**Comment by colin-gannon** [comment]
Thanks so much! I’ll give it a whirl later this week and see how I do with getting it built and stood up. Great work on crackq – simple but effective (just how I like it)!

From: f0cker ***@***.***>
Date: Sunday, July 2, 2023 at 1:25 PM
To: f0cker/crackq ***@***.***>
Cc: Colin Gannon ***@***.***>, Author ***@***.***>
Subject: Re: [f0cker/crackq] Front end repo access (Discussion #34)
This is done now, see here: https: //github. com/f0cker/crackvue I should have some time to start pushing some updates soon. Any problems give me a shout on the Discord channel. — Reply to this email directly, view it on GitHub, or unsubscribe. You


This is done now, see here: https://github.com/f0cker/crackvue<https://urldefense.com/v3/__https:/github.com/f0cker/crackvue__;!!HWWtag!yiyWyKzm4Kp9uceLom5Pcj4xl-ciHEOqA_yfTkO5s5bX-i4cPapZxpH0ZCtYR3o2WOtQv9j_XHaAIeIX_lLAhfCz$>

I should have some time to start pushing some updates soon. Any problems give me a shout on the Discord channel.

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/f0cker/crackq/discussions/34*discussioncomment-6337827__;Iw!!HWWtag!yiyWyKzm4Kp9uceLom5Pcj4xl-ciHEOqA_yfTkO5s5bX-i4cPapZxpH0ZCtYR3o2WOtQv9j_XHaAIeIX_lDgN56I$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/A3GSKXMX3HTPUKAUAPU7UQLXOGVGPANCNFSM6AAAAAAZYRKORQ__;!!HWWtag!yiyWyKzm4Kp9uceLom5Pcj4xl-ciHEOqA_yfTkO5s5bX-i4cPapZxpH0ZCtYR3o2WOtQv9j_XHaAIeIX_gkIqs84$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

===============================================================================
Please refer to http://www.kkr.com/email-disclaimer for important disclosures regarding this electronic communication.
KKR uses your personal information in accordance with its privacy notices, which can be found on http://www.kkr.com.
===============================================================================
